tabpy-server/server_tests/test_config.py

```diff
@@ -70,24 +92,25 @@ class TestPartialConfigFile(unittest.TestCase):
     def test_config_file_present(self, mock_os, mock_path_exists, mock_psws, mock_shutil, mock_management_util,
                                  mock_tabpy_state, mock_parse_arguments):
         config_file = NamedTemporaryFile(delete=False)
-        config_file.write(b'TABPY_BIND_IP = 0.0.0.0\n'
-                          b'TABPY_QUERY_OBJECT_PATH = foo\n'
-                          b'TABPY_STATE_PATH = bar\n'
-                          b'TABPY_LOG_LEVEL = warning')
+        append_logger_settings_to_config_file(config_file)
+
+        config_file.write("[TabPy]\n"
```

Comment:
So assuming you don't go through with my previous suggestion of using io instead of a file handle, you should at the very least probably use a context manager here for the tempfile

Reply:
I agree it looks weird but at the moment I am concentrated on making this work